index.html

```diff
@@ -51,14 +53,14 @@ <h3>Generic Use-Case</h3>
       <section id="practical-use-case">
         <h3>Practical Use-Case</h3>
         <p>
-          Consider a combo-application consisting of two major parts - a video-conferencing
-          application and a productivity-suite application co-existing in a [=browsing context=].
-          Assume the video-conferencing uses existing/upcoming APIs such as
-          {{MediaDevices/getDisplayMedia()}} and/or <code>getViewportMedia</code> and captures the
-          entire [=browsing context=]. Now it needs to crop away everything other than a particular
-          section of the productivity-suite. It needs to crop away its own video-conferencing
-          content, any speaker notes and other private and/or irrelevant content in the
-          productivity-suite, before transmitting the resulting cropped video remotely.
+          Consider a combo-application consisting of two major parts living in one tab - a
```

Comment:
With layterms we can scope this to the page I think
```suggestion
          Consider a combo-application consisting of two major parts in different iframes - a
```

Reply:
How about "different iframes of a single tab"?

Reply:
I'm going to commit your suggestion then apply the addition, @jan-ivar.